application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -141,4 +141,13 @@ public static String abbreviate(String text, int maxLength) {
         return text.substring(0, maxLength - ABBREVIATION.length()) + ABBREVIATION;
     }
 
+    /**
+     * Mentions a guild channel by its id
```

Comment:
* missing dot.
* missing the behavior when given an unknown ID

Reply:
(not done yet)

Reply:
> missing the behavior when given an unknown ID

still not done yet.

Reply:
done